lib/serialize-value.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var destroyCircular = require('destroy-circular');
+
+// Make a value ready for JSON.stringify() / process.send()
+
+module.exports = function serializeValue(value) {
+	if (typeof value === 'object') {
+		return destroyCircular(value);
+	}
+	if (typeof value === 'function') {
+		// JSON.stringify discards functions
```

Comment:
TDIL:
```
let foo = function () {};
let bar = foo;

console.log(bar.name);
// => foo
```

![the more you know](http://media.giphy.com/media/Y2nbrJyAR6RiM/giphy-facebook_s.jpg)